tests/components/main-panel/main-panel.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { MainPanel } from "src/components";
+import { EElementType } from "src/context-providers";
+import { ELEMENT_BUTTON_LABELS } from "src/data/elements-data";
+import { TestHelper, mockBuilderState } from "src/util/test-helper";
+
+describe("main-panel", () => {
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+
+    describe("conditional rendering of main panel content", () => {
+        it("should display the empty panel content", async () => {
+            renderComponent();
+            await expect(getEmptyPanelContent()).resolves.toBeInTheDocument();
```

Comment:
Do you need the await? I don't think there are any state changes?

Reply:
when i remove the async i get this error 
![image](https://github.com/LifeSG/web-form-builder/assets/103546625/47e73602-4a0e-49d2-93c2-7ceb31df824b)